tests/softlearning/environments/adapters/softlearning_env_test.py

```diff
@@ -1,6 +1,6 @@
 import unittest
 import numpy as np
-
+from collections import OrderedDict
```

Comment:
This should go between the unit test and numpy import. Generally, the order of imports should be

```
<python standard libraries>

<non-standard libraries>

<project libraries>
```

I should create a linter that automatically formats the files, but for now I would just add some linter to you sublime text that automatically converts the stuff for you.

Reply:
Fixed. DMed you about a linter plugin.